cves/CVE-2010-1157.yml

```diff
@@ -105,12 +102,9 @@ unit_tested:
     For the "fix" answer below, check if the fix for the vulnerability involves
     adding or improving an automated test to ensure this doesn't happen again.
     Must be just "true" or "false".
-  answer: |
-    From looking at the git log, the code involved in the commits, and the bug reports,
-    I could not determine that any unit testing or automated testing was used on this
-    section of the code.
-  code: false
-  fix: false
+  answer: false
+  code:
+  fix:
```

Comment:
I believe both the "fix" and "code" fields must be either true or false, as stated in the question above.

Reply:
You are correct. I have fixed this.